app-backup/snapper/snapper-0.5.4.ebuild

```diff
@@ -47,13 +44,15 @@ src_prepare() {
 }
 
 src_configure() {
+	# ext4 code does not work anymore
+	# we need btrfs for snapper to work
```

Comment:
This sentence does not clearly catch the problem. If it doesn't build with btrfs disabled, you should just say that.

Reply:
ok